### PR TITLE
Bah 2422

### PR DESCRIPTION
**Pull request by manimaarans**

### CODE REVIEW

File: openmrs/apps/customDisplayControl/views/printCertificate.html [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+            <span class="label">Date:</span>
+            <span>{{today | date:'dd/MM/yyyy'}} </span> </br>
+            <span class="label">Patient Registration No:</span>
+            <span>{{patient.identifier}}</span>
+        </div>
+        <div>
+            <font size="3">
+                <h3 style="text-align: center; margin-bottom: 20px">WHOM IT MAY CONCERN</h3>
+                <p style="text-align: justify;">This is to inform that<b> {{patient.givenName}} {{patient.middleName}}
+                    {{patient.familyName}}</b> Age: {{patient.age || "0"}} years, is undergoing treatment for
```
Comment (gsluthra) [inline openmrs/apps/customDisplayControl/views/printCertificate.html]: This all seems to be hardcoded in English. Shouldn't this whole statement come from translation file?

File: openmrs/apps/customDisplayControl/views/printCertificate.html [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+    <hr>
+
+    <div>
+        <span class="label"><h3 style="text-align: center;">Fitness Certificate</h3></span>
+    </div>
+    <div>
+        <div style="text-align: left;">
+            <span class="label">Date:</span>
+            <span>{{today | date:'dd/MM/yyyy'}} </span> </br>
+            <span class="label">Patient Registration No:</span>
```
Comment (gsluthra) [inline openmrs/apps/customDisplayControl/views/printCertificate.html]: Shouldn't this come from translation file?

File: openmrs/apps/customDisplayControl/views/printCertificate.html [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+            <span>{{patient.identifier}}</span>
+        </div>
+        <div class="content-body">
+            <font size="3">
+                <h3 style="text-align: center;">WHOM IT MAY CONCERN</h3>
+                <p style="text-align: justify;">This is to inform that<b> {{patient.givenName}} {{patient.middleName}}
+                    {{patient.familyName}}</b> Age: {{patient.age || "0"}} years, was undergoing treatment for
```
Comment (gsluthra) [inline openmrs/apps/customDisplayControl/views/printCertificate.html]: This all seems to be hardcoded in English. Shouldn't this whole statement come from translation file?

File: openmrs/apps/customDisplayControl/views/printCertificate.html [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+                    <span treatment patient="patient">{{formFieldValues[0]["Treatment plan"]}}</span>.
+                    {{patient.givenName}} {{patient.middleName}}
+                    {{patient.familyName}} is now fit to resume daily
+                    activities. </p>
+                <p class="signature" style="margin-top: 20px">Regards,</p>
+                <p class="signature"><b>{{doctorName}},</b></p>
+                <p class="signature">Reg.No. {{registrationNumber}}. </p>
```
Comment (gsluthra) [inline openmrs/apps/customDisplayControl/views/printCertificate.html]: Shouldn't this  come from translation file?

**Comment by manimaarans** [comment]
@gsluthra we have added the feedback changes and updated the pr. can you please review the PR?

File: openmrs/i18n/clinical/locale_en.json [hunk 5]
```diff
@@ -86,5 +86,23 @@
...
+
+  "DASHBOARD_TAB_PRINT": "Print",
+  "UNDER_TREATMENT_CERTIFICATE": "Under Treatment Certificate",
+  "FITNESS_CERTIFICATE": "Fitness Certificate",
+  "NAVIGATE_TO_UNDER_TREATMENT_CERTIFICATE_FORM" : "Consultation > Observation forms > Under Treatment and Fitness Certificate.",
+  "PRINT_BUTTON": "Print",
+  "DATE_LABEL": "Date:",
+  "PATIENT_REGISTRATION_NO_LABEL": "Patient Registration No:",
+  "CERTIFICATE_SUBJECT_1": "WHOM IT MAY CONCERN",
+  "CERTIFICATE_SUBJECT_2": "This is to inform that",
```
Comment (gsluthra) [inline openmrs/i18n/clinical/locale_en.json]: This is not going to work. In different languages, the sentence construction is not going to be the same.
The whole email body (or paragraph) needs to be ONE SENTENCE... with parameters. 

Read this: https://ux.shopify.com/how-to-concatenate-localized-strings-mindfully-bcc7b6d4961a

The verbs/nouns are placed in different locations in different languages. Hence we should not use concatenation. We should use interpolation. https://en.wikipedia.org/wiki/String_interpolation

Comment (gsluthra) [reply openmrs/i18n/clinical/locale_en.json]: CERTIFICATE_BODY_PARA1 = "full string/sentence"
CERTIFICATE_BODY_PAR2 = "full string/sentence"

Comment (manimaarans) [reply openmrs/i18n/clinical/locale_en.json]: @gsluthra we have updated the pr with feedback changes. can you please review the changes?

**Review by gsluthra — APPROVED** [review]

**Pull request merged**